transaction_integration_test.go

```diff
@@ -657,6 +657,7 @@ func TestAllTransactionFields(t *testing.T) {
 			StoreInVault:                     true,
 			AddBillingAddressToPaymentMethod: true,
 			StoreShippingAddressInVault:      true,
+			SkipAdvancedFraudChecking:        true,
```

Comment:
This line is causing the test to fail because there are other expectations on the response returned that are not present when you set this flag.